### ISSUE TITLE
Including other queries (includeTimeline) with getMatch

### ISSUE THREAD

**Issue by sethlei** [kevinohashi/php-riot-api#48]
Currently, this api does not offer an obvious way to request timeline events from the riot games api. Could a flag be added to include this?

**Comment by sethlei** [comment]
These are the changes I made to make it work, but they're probably very novice.

public function getMatch($matchId, $otherqueries) {
		$call = self::API_URL_2_2  . 'match/' . $matchId;
		return $this->request($call, $otherqueries);
	}

	private function request($call, $otherQueries, $static = false) {
				//format the full URL
		$url = $this->format_url($call, $otherQueries);

	private function format_url($call, $otherQueries){
		//because sometimes your url looks like .../something/foo?query=blahblah&api_key=dfsdfaefe
		if (!$otherQueries){
		return str_replace('{region}', $this->REGION, $call) . ($otherQueries ? '&' : '?') . 'api_key=' . self::API_KEY;
		}
		else {
		return str_replace('{region}', $this->REGION, $call) . $otherQueries . '&api_key=' . self::API_KEY;
		}
	}

**Comment by kevinohashi** [comment]
Just thinking about this, there already is an $otherQueries option in request() function. So just adding $otherqueries to getMatch seems like it would work in your first function. Is that what you did or where did you have issues?

**Comment by sethlei** [comment]
$otherqueries was set to false in the method call, and seemed to overwrite anything I would pass in. I added some logic in the format url to preserve functionality of the other method calls which only use one input. 

Also my $otherqueries = "?includeTimeline=true" because the question mark isn't added. I'll probably clean this whole section up later - but for now I've moved onto working on the logic I am using the API for.
